tls/s2n_quic_support.h

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
I think this file needs to go in the [`api`](https://github.com/awslabs/s2n/tree/main/api) directory so it's publicly consumable.

Reply:
I mention this in the PR description. This file definitely needs to go into `api` eventually, but I'm hesitant to put it there when it's a totally incomplete feature. I think we should move the APIs there once S2N is useable by QUIC as a TLS implementation (probably still before the APIs are finalized).

Reply:
We talked offline: this is OK for now